src/applications/vaos/services/appointment/transformers.js

```diff
@@ -340,28 +341,28 @@ function getCommunityCareData(appt) {
  */
 function setLocation(appt) {
   const type = getAppointmentType(appt);
+  const location = {
+    vistaId: null,
+    stationId: null,
+    clinicId: null,
+    clinicName: null,
+  };
 
-  switch (type) {
-    case APPOINTMENT_TYPES.vaAppointment: {
-      return {
-        vistaId: appt.facilityId,
-        clinicId: appt.clinicId,
-        stationId: appt.sta6aid,
-        clinicName:
-          appt.clinicFriendlyName ||
-          appt.vdsAppointments?.[0]?.clinic?.name ||
-          appt.vvsAppointments?.[0]?.patients?.[0]?.location?.clinic?.name,
-      };
-    }
-    case APPOINTMENT_TYPES.request: {
-      return {
-        vistaId: appt.facility?.facilityCode?.substring(0, 3),
-        stationId: appt.facility?.facilityCode,
-      };
-    }
-    default:
-      return {};
+  if (type === APPOINTMENT_TYPES.vaAppointment) {
```

Comment:
Had to change this to an if because we no longer had a default case for the switch